packages/manager/apps/web-office/src/pages/onboarding/index.tsx

```diff
@@ -1,13 +1,17 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { useTranslation } from 'react-i18next';
-import { OnboardingLayout } from '@ovh-ux/manager-react-components';
+import { Card, OnboardingLayout } from '@ovh-ux/manager-react-components';
+import { ShellContext } from '@ovh-ux/manager-react-shell-client';
 import { Breadcrumb } from '@/components/Breadcrumb/Breadcrumb';
 import onboardingImgSrc from './onboarding-img.png';
+import { GUIDES_LIST } from '@/guides.constants';
 
 export default function Onboarding() {
   const { t } = useTranslation('onboarding');
-  const title: string = t('title');
-  const description: string = t('description');
+  const context = useContext(ShellContext);
+  const { ovhSubsidiary } = context.environment.getUser(); // Récupère la filiale de l'utilisateur
```

Comment:
```suggestion
  const { ovhSubsidiary } = context.environment.getUser();
```
I'm not sure that needed this comment